### ISSUE TITLE
Ability to hide / single layers

### ISSUE THREAD

**Issue by tobbi** [SuperTux/supertux#517]
The C# editor has this and I think it's quite a useful feature:

Hiding layers or displaying one layer only helps people designing levels quite a bit, IMO.

Not sure if we want this for 0.5.0 or later, though.

This was the eye symbol in the old editor.

**Comment by maxteufel** [comment]
Let's keep this for a later release. We also do not want to overbloat the in-game editor. Flexlay's existence has a reason as well.

**Comment by Karkus476** [comment]
@maxteufel I agree, although this is an almost essential feature to make more aesthetically pleasing levels possible in the editor

**Comment by maxteufel** [comment]
@Karkus476 for now, non-selected layers have a decreased opacity anyway.

**Comment by sofi004** [comment]
In addiction to this can i also do something so that the users can choose the level of opacity in every layer?

**Comment by tobbi** [comment]
Hello, @sofi004,

@Hypernoot has assigned the issue to you. If you have any questions, feel free to ask them in this thread.

**Comment by Alasdairbugs** [comment]
I believe this exists by using the fill option
